### PR TITLE
add: solution to project euler problem 20

### PR DESCRIPTION
**Pull request by kogoking**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

Adding solution for Project Euler problem 20 md file.

### CODE REVIEW

File: curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-20-factorial-digit-sum.english.md [hunk 1]
```diff
@@ -61,7 +61,16 @@ sumFactorialDigits(100);
...
+let factorial = (n) => n <= 1 ? BigInt(n) : BigInt(n) * BigInt(factorial(--n));
+
+let sumDigits = n => n.toString().split('').map(x => parseInt(x)).reduce((a,b) => a + b);
+
+function sumFactorialDigits(n)
+{
+  return sumDigits(factorial(n));
+}
+
+sumFactorialDigits(100);
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-20-factorial-digit-sum.english.md]:
```suggestion
```
This isn't required and makes the solution a little slower.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Small nitpick, but this looks good otherwise.

**Comment by kogoking** [comment]
> Small nitpick, but this looks good otherwise.

Thanks for pointing it out. Do I only need to commit now? Contributing for the first time.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Comment by ojeytonwilliams** [comment]
>  Do I only need to commit now? 

Adding an extra commit that removes the line is perfect, thanks!

File: curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-20-factorial-digit-sum.english.md [hunk 2]
```diff
@@ -61,7 +61,15 @@ sumFactorialDigits(100);
 <section id='solution'>
 
 ```js
-// solution required
+let factorial = (n) => n <= 1 ? BigInt(n) : BigInt(n) * BigInt(factorial(--n));
+
+let sumDigits = n => n.toString().split('').map(x => parseInt(x)).reduce((a,b) => a + b);
+
+function sumFactorialDigits(n)
```
Comment (Manish-Giri) [inline curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-20-factorial-digit-sum.english.md]:
```suggestion
function sumFactorialDigits(n) {
```

File: curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-20-factorial-digit-sum.english.md [hunk 3]
```diff
@@ -61,7 +61,15 @@ sumFactorialDigits(100);
...
+{
```
Comment (Manish-Giri) [inline curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-20-factorial-digit-sum.english.md]:
```suggestion

```

**Review by Manish-Giri — CHANGES_REQUESTED** [review]
More nitpicking (sorry!), but a minor style change...

**Review by Manish-Giri — APPROVED** [review]
LGTM. Congratulations on your first pull request 💯

**Pull request merged**